guides/base-and-override-templates.md

```diff
@@ -5,3 +5,47 @@ permalink: /guides/base-and-override-templates/
 ---
 
 #Base and Override Templates
+
+An important need in developing a multi-page web application is to have common elements of the pages defined just once and shared by all pages (DRY=Don't Repeat Yourself). Dust provides this with the concept of blocks, base, and override templates. Consider a common case where several pages share a header and footer but have different body content.
```

Comment:
"... with the concept of blocks, inline-partials, and base and override templates. ..."